bundles/org.openhab.binding.knx/src/main/resources/OH-INF/thing/ip.xml

```diff
@@ -64,6 +72,32 @@
 				<description>Seconds between connection retries when KNX link has been lost, 0 means never retry, minimum 30s</description>
 				<default>60</default>
 			</parameter>
+			<parameter name="routerBackboneKey" type="text" groupName="knxsecure">
+				<context>password</context>
+				<label>Router backbone key</label>
+				<description>KNX secure, optional: Backbone key for secure router mode. 16 bytes
```

Comment:
Please remove the "KNX secure, optional:" from all parameters - they anyhow appear in the "KNX secure" parameter group.